Assets/HoloToolkit/Input/Scripts/Voice/SpeechInputSource.cs

```diff
@@ -0,0 +1,181 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+using UnityEngine.Windows.Speech;
+
+namespace HoloToolkit.Unity.InputModule
+{
+    /// <summary>
+    /// SpeechInputSource allows you to specify keywords and methods in the Unity
+    /// Inspector, instead of registering them explicitly in code.
+    /// This also includes a setting to either automatically start the
+    /// keyword recognizer or allow your code to start it.
+    ///
+    /// IMPORTANT: Please make sure to add the microphone capability in your app, in Unity under
+    /// Edit -> Project Settings -> Player -> Settings for Windows Store -> Publishing Settings -> Capabilities
+    /// or in your Visual Studio Package.appxmanifest capabilities.
+    /// </summary>
+    public partial class SpeechInputSource : BaseInputSource
```

Comment:
No worries, yeah I agree we should skip the other review content and get back to it later.